packages/compartment-mapper/src/policy.js

```diff
@@ -0,0 +1,64 @@
+// @ts-check
+
+const { keys, create, entries } = Object;
+
+const { has } = Reflect;
+
+const copyGlobals = (globals, list) => {
```

Comment:
To be replaced by https://github.com/LavaMoat/LavaMoat/blob/main/packages/core/src/makeGetEndowmentsForConfig.js

Reply:
`__isKnownScopeProxy__` should no longer be an issue.